Source/gg2/Objects/Overlays/Gamemodes/ControlPointHUD.events/Create.xml

```diff
@@ -21,8 +21,10 @@ global.totalControlPoints = instance_number(ControlPoint);
 
 if instance_exists(ControlPointSetupGate)
 {
-    global.setupTimer=1800;
+    global.setupTimer=900;
     mode=1;
+    // 3 minutes
+    timeLimit = 3*30*60;
```

Comment:
>Besides, timer is set to timeLimit above this line, so will we actually start with the correct timer value?

seems to work
![asdsdf](http://i.imgur.com/kBd4Xep.png)

>This overwrites a time limit that could previously be set as an option. Is it really ok to have that setting work for all cp modes except adcp?

gameplay wise, having the same (usually long) time limit for adcp makes the game drag way more than it should